language/documentation/hackathon-tutorial/README.md

```diff
@@ -18,15 +18,21 @@ $ sh step_0/setup.sh
 $ . ~/.profile
 ```
 
-Once this is done, you can alias the `move` package command to `mpm`:
+Once this is done, you can alias the `move` command to point the `df-cli`
```

Comment:
Good question, I don't think this should cause too much of an issue since the only real difference between the df-cli and move binaries is the set of natives (with df-cli having a superset of the move natives). It may be worth polling the room for a good name for this at the content discussion later today.